ci/Dockerfile

```diff
@@ -1,39 +1,46 @@
 # vim: set expandtab shiftwidth=4 softtabstop=4:
-FROM alpine:3.10
+FROM alpine:3.12
 
 RUN apk add --no-cache --no-progress --update --upgrade \
-    bash \
-    bridge-utils \
-    coreutils \
-    cpio \
-    curl \
-    diffutils \
-    dnsmasq \
-    docker \
-    eudev \
-    file \
-    findutils \
-    git \
-    go \
-    gptfdisk \
-    iptables \
-    jq \
-    libarchive-tools \
-    lshw \
-    make \
-    musl-dev \
-    ovmf \
-    pigz \
-    py2-pip \
-    qemu-system-aarch64 \
-    qemu-system-x86_64 \
-    rsync \
-    socat \
-    util-linux \
-    python3 \
-    python3-dev \
-    gcc \
-    && apk add --no-cache --update --upgrade \
+        bash \
+        bridge-utils \
+        coreutils \
+        cpio \
+        curl \
+        diffutils \
+        dnsmasq \
+        docker \
+        eudev \
+        file \
+        findutils \
+        gcc \
+        git \
+        go \
+        gptfdisk \
+        iptables \
+        jq \
+        libarchive-tools \
+        make \
+        musl-dev \
+        ovmf \
+        pigz \
+        py3-pip \
+        python3 \
+        python3-dev \
+        qemu-system-aarch64 \
+        qemu-system-x86_64 \
+        rsync \
+        socat \
+        util-linux \
+        && \
+    mv /etc/apk/repositories /etc/apk/repositories.old && \
+    apk add --no-cache --update --upgrade \
+        --repository=http://dl-cdn.alpinelinux.org/alpine/v3.11/main \
```

Comment:
I think this needs to be changed to `v3.12` (maybe do a global search and replace just to check whether this does not appear elsewhere).

Reply:
no this is to get lshw from 3.11 explicitly. v3.12's `lshw` doesn't output correct json. I have not been able to track down the commit that broke `lshw`.

Reply:
Oh I see :-(

Reply:
Its only used in tests so not a big deal imo. Looks like upstream has fixed the json output though so I'll look into PR'ing patches to aports.

Reply:
https://gitlab.alpinelinux.org/alpine/aports/-/merge_requests/11607 in case you're interested @rgl